From: Michael Grzeschik <m.grzeschik@pengutronix.de>
Date: Mon, 11 Jul 2011 17:18:40 +0200
Subject: [PATCH] Disable build of qv4l2

* currently disable support for qv4l2 with
  a patch on the Makefile. Otherwise qmake
  will not find the v4l libraries.

This is not for upstream!

---
 utils/Makefile |   15 ---------------
 1 files changed, 0 insertions(+), 15 deletions(-)

diff --git a/utils/Makefile b/utils/Makefile
index 71a3cb7..0cb03e1 100644
--- a/utils/Makefile
+++ b/utils/Makefile
@@ -10,21 +10,6 @@ all install:
 		$(MAKE) -C v4l2-sysfs-path $@; \
 	fi
 
-	# Test whether qmake is installed, and whether it is for qt4.
-	@if which qmake-qt4 >/dev/null 2>&1; then \
-		QMAKE=qmake-qt4; \
-	else \
-		QMAKE=qmake; \
-	fi; \
-	if which $$QMAKE >/dev/null 2>&1; then \
-		if $$QMAKE --version 2>&1 | grep '4\.[0-9][0-9]*\.[0-9][0-9]*' >/dev/null; then \
-			if [ ! -f qv4l2/Makefile ]; then \
-				(cd qv4l2 && $$QMAKE) || exit 1; \
-			fi; \
-			$(MAKE) -C qv4l2 -f Makefile.install $@; \
-		fi \
-	fi
-
 sync-with-kernel:
 	$(MAKE) -C keytable $@
 	$(MAKE) -C v4l2-dbg $@
-- 
1.7.5.4

